api/goals.py

```diff
@@ -61,4 +74,63 @@ def post(self):
             return make_response(jsonify(responseObject)), 401
 
 
+
+class CheckGoalsAPI(MethodView):
+    """
+    User Reservations
+    """
+    def post(self):
```

Comment:
Please use `@authorize` in this and the API above. Without it, you won't be able to access `g.user.id`. Also makes sure you import everything required for using the decorator.

Reply:
resolved